frontend/pages/admin/IntegrationsPage/cards/Mdm/components/RequestCSRModal/_styles.scss

```diff
@@ -0,0 +1,38 @@
+.request-csr-modal {
+  &__info-header {
+    margin-bottom: $pad-xlarge;
+  }
+
+  &__form {
+    .bottom-label {
+      .form-field {
+        margin-bottom: 2px;
+      }
+
+      p {
+        margin-top: 0px;
+        color: $ui-fleet-black-75;
+      }
+    }
+  }
+
+  .check {
+    border-radius: 50%;
+    border-color: $core-fleet-blue;
+    border: solid;
+    padding: 5px;
+  }
+
+  .success {
+    text-align: center;
+
+    svg {
```

Comment:
Let's remove the styling on this SVGs. It looks like you are trying to control the size of the SVG which can be done with a `size` variant prop on the`<Icon />` component. check out `ExternalLink.tsx` for an example how.